src/main/java/ama/account/User.java

```diff
@@ -27,6 +26,12 @@
     @NotEmpty
     private String password;
 
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Is this necessary because of LazyInitializationException during tests? If so, I believe you could add @Transactional above the test classes and omit this here. That way the collection would not need to always be fetched and only need be fetched on-demand.